src/components/existing-request/existing-request-display.vue

```diff
@@ -250,20 +249,15 @@ export default class ExistingRequestDisplay extends Mixins(
   }
 
   /** This is True while refreshing the NR. (Not used in template at the moment.) */
-  private checking = false
+  checking = false
```

Comment:
VS Code tells me this property isn't used if it's private. (It's actually used in the template.)